src/pages/ActivityCollectionAdminPage.tsx

```diff
@@ -39,6 +38,8 @@ const ActivityCollectionAdminPage: React.FC = () => {
           <div className="mb-5">
             <ProductCreationModal
               categoryClassType="activity"
+              creatorAppellation={formatMessage(commonMessages.label.selectOrganizer)}
+              allowedPermissions={['ACTIVITY_NORMAL']}
```

Comment:
Is `ACTIVITY_ADMIN` also needed?

Reply:
You're right, I will add it in the next commit, thanks for catching my oversight.